Readme.md

```diff
@@ -58,6 +58,10 @@ Once you've read the overview of OpenLibrary Backend technologies, it's highly e
 
 - [Infogami Developer Tutorial](https://openlibrary.org/dev/docs/infogami)
 
+After reading the tech and developments for a background on Infogami, the most recent files can be viewed and worked on.
+
+- [Infogami master files](https://github.com/internetarchive/infogami)
```

Comment:
```suggestion
If you want to dive into the source code for Infogami, see the [Infogami repo](https://github.com/internetarchive/infogami).
```

- Removed the bullet point to reduce visual weight; this is less essential for OL development than the other 2 points.
- Used `If ...` to clarify this is an optional, not essential step of getting familiar with Open Library's backend.

How's this @cclauss @BrittanyBunk ?

Reply:
Perfect!